platform/node/src/node_map.cpp

```diff
@@ -278,9 +278,8 @@ void NodeMap::renderFinished() {
             [](char *, void *hint) {
                 delete reinterpret_cast<const mbgl::StillImage *>(hint);
             },
-            const_cast<mbgl::StillImage *>(img.get())
+            const_cast<mbgl::StillImage *>(img.release())
         ).ToLocalChecked();
-        img.release();
```

Comment:
Reverted in https://github.com/mapbox/mapbox-gl-native/commit/6d671db02a1425cb11d06cc40facf7944e27e812